R/insertPackage.R

```diff
@@ -130,7 +130,7 @@ identifyPackageType <- function(file) {
     ret <- if (grepl("_.*\\.tar\\..*$", file)) {
         "source"
     } else if (grepl("_.*\\.tgz$", file)) {
-        "mac.binary"
+        "mac.binary.mavericks"
```

Comment:
That is exactly what I said: as far as the PACKAGES format goes, there is one type for OS X and one type for Windows.